docs/submitting-clinical-data.md

```diff
@@ -2,7 +2,7 @@
 id: submitting-clinical-data
 title: Submitting Clinical Data
 ---
-Clinical data consists of all supporting data about your program's donors and their health data.
+This guide will describe how to submit clinical data to the ARGO Data Platform. Clinical data consists of all supporting data about your program's donors and their health data.
```

Comment:
Clinical data consists of all supporting data about your program's donors including their health data.